tests/SampleApps/nodejs/kudu-bug/another-directory/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "kudu-bug",
```

Comment:
please rename this app to be more appropriate for the scenario being tested here...something which indicates `nested package.json`